### PR TITLE
Add clarifying doc about node transformation

### PR DESCRIPTION
**Pull request by lilleyse**
For https://github.com/AnalyticalGraphicsInc/cesium/issues/7615

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @lilleyse!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/node-transforms-doc/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

File: Source/DataSources/ModelGraphics.js [hunk 1]
```diff
@@ -201,6 +201,7 @@ define([
         /**
          * Gets or sets the set of node transformations to apply to this model.  This is represented as an {@link PropertyBag}, where keys are
          * names of nodes, and values are {@link TranslationRotationScale} Properties describing the transformation to apply to that node.
+         * The transformation is applied before the node's existing transformation as specified in the glTF, and does not replace the node's existing transformation.
```
Comment (emackey) [inline Source/DataSources/ModelGraphics.js]: before?  or after?

Comment (emackey) [reply Source/DataSources/ModelGraphics.js]:
```
modelNode.matrix = Matrix4.multiply(modelNode.originalMatrix, transformationMatrix, transformationMatrix);
```

Pretty sure it's after...

Comment (lilleyse) [reply Source/DataSources/ModelGraphics.js]: Thanks.. fixed.

**Pull request merged**